cypress/support/elements/clue/ChatPanel.js

```diff
@@ -61,10 +61,12 @@ class ChatPanel{
       return cy.get('[data-testid=tool-tile]').eq(tileIndex);
     }
     typeInCommentArea(commentText) {
-      cy.get("[data-testid=comment-textarea]").type(commentText);
+      // If the comment list is long, the text box is off screen so force typing.
+      cy.get("[data-testid=comment-textarea]").type(commentText, {force:true});
```

Comment:
try some of the other options that aren't as heavy handed.

Reply:
Actually fixed by removing an unnecessary div that was messing up the scroll behavior.